src/routes/session/session-get.ts

```diff
@@ -1,18 +1,41 @@
-import { NextFunction, Request, Response } from "express";
+import { Request, Response } from "express";
 import { createUrlWithQueryString } from "utils/create-url-with-query-string";
 
-export const SessionGet = (req: Request, res: Response, next: NextFunction) => {
-	if (!req.params[0]) {
-		return next(new Error("Missing redirect url for session.  Needs to be in format `/session/:redirectUrl`"));
-	}
-
-	const url = createUrlWithQueryString(req, req.params[0]);
+export const SessionGet = (req: Request, res: Response) => {
+	const url = createUrlWithQueryString(req, req.params[0] || "");
+	const { title, loadAutoAppCreation } = configForPage(req);
 
-	return res.render("session.hbs", {
-		title: "Logging you into GitHub",
-		APP_URL: process.env.APP_URL,
+	res.render("session.hbs", {
 		redirectUrl: new URL(url, process.env.APP_URL).href,
```

Comment:
What are we expecting to happen here when url is an empty string? Is there a reason we ditched the error that @mboudreau had added here?